src/System.Net.Sockets/src/System/Net/Sockets/TCPClient.cs

```diff
@@ -72,6 +72,22 @@ protected bool Active
 
         public int Available { get { return AvailableCore; } }
 
+        // Used by the class to provide the underlying network socket.
+        [DebuggerBrowsable(DebuggerBrowsableState.Never)] // TODO: Remove once https://github.com/dotnet/corefx/issues/5868 is addressed.
```

Comment:
I don't understand why this attribute is here. I thought we were adding back the .Client property. Why would we mark it as `DebuggerBrowseableState.Never` ?